### PR TITLE
go-mgo is unmaintained

### PR DESCRIPTION
**Pull request by nawa**
See author's last commit
https://github.com/go-mgo/mgo/commit/b86855e89b5a6a30ea0c31ef8bb16bf17179734d

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,14 +369,15 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
...
     * [go-couchbase](https://github.com/couchbase/go-couchbase) - Couchbase client in Go.
     * [go-couchdb](https://github.com/fjl/go-couchdb) - Yet another CouchDB HTTP API wrapper for Go.
     * [gocb](https://github.com/couchbase/gocb) - Official Couchbase Go SDK.
     * [gocql](http://gocql.github.io) - Go language driver for Apache Cassandra.
     * [gomemcache](https://github.com/bradfitz/gomemcache/) - memcache client library for the Go programming language.
     * [gorethink](https://github.com/dancannon/gorethink) - Go language driver for RethinkDB.
     * [goriak](https://github.com/zegl/goriak) - Go language driver for Riak KV.
-    * [mgo](https://godoc.org/labix.org/v2/mgo) - MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms.
+    * [mgo](https://godoc.org/labix.org/v2/mgo) - UNMAINTAINED. MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms
```
[the remark this answers is not in the document]
Comment (nawa) [reply README.md]: @appleboy removed

File: README.md [hunk 2]
```diff
@@ -376,7 +376,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
     * [gomemcache](https://github.com/bradfitz/gomemcache/) - memcache client library for the Go programming language.
     * [gorethink](https://github.com/dancannon/gorethink) - Go language driver for RethinkDB.
     * [goriak](https://github.com/zegl/goriak) - Go language driver for Riak KV.
-    * [mgo](https://godoc.org/labix.org/v2/mgo) - MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms.
+    * [mgo] (https://godoc.org/github.com/globalsign/mgo) MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms
```
Comment (appleboy) [inline README.md]: remove space between `[mgo]` and `(https://xxxx)`

Comment (nawa) [reply README.md]: @appleboy oh

completed

File: README.md [hunk 3]
```diff
@@ -376,7 +376,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
     * [gomemcache](https://github.com/bradfitz/gomemcache/) - memcache client library for the Go programming language.
     * [gorethink](https://github.com/dancannon/gorethink) - Go language driver for RethinkDB.
     * [goriak](https://github.com/zegl/goriak) - Go language driver for Riak KV.
-    * [mgo](https://godoc.org/labix.org/v2/mgo) - MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms.
+    * [mgo](https://godoc.org/github.com/globalsign/mgo) MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms
```
Comment (appleboy) [inline README.md]: maybe change the URL to https://github.com/globalsign/mgo

Comment (nawa) [reply README.md]: no difference, but as you wish

**Comment by appleboy** [comment]
@nawa Thanks LGTM

**Pull request merged**